server/db/svarog-10.psql

```diff
@@ -0,0 +1,19 @@
+-- This schema version introduces station ownership relation.
+-- For each station, there can be an arbitrary number of users that are owners.
+-- In the future, the owner will be able to some administrative actions, such as
+-- deleting observation, edit station details etc.
+
+CREATE TABLE station_owners (
```

Comment:
Use PRIMARY KEY constraint and define "user_id" + "station_id" as a key of this table.

Reply:
Done.